cafeteria/src/main/java/org/itt/utility/Client.java

```diff
@@ -0,0 +1,42 @@
+package org.itt.utility;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.net.Socket;
+
+public class Client {
```

Comment:
Why are your client, server etc. in utility folder, as these are the entry points to the app?